packages/conf-libpcre2-8/conf-libpcre2-8.2/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
```

Comment:
What does the `-8` mean in the package name?

Reply:
There is an explanation for the library name in: #21349

Reply:
Got it, thanks! Sorry again that I missed the context of the previous release.